### DISCUSSION TITLE
Explain the Confusion Matrix with Respect to Machine Learning Algorithms.

### DISCUSSION THREAD

**Discussion by luffy-Monkey123** [luffy-Monkey123/python-cool-programs#17] [Q&A] [answered]
Explain the Confusion Matrix with Respect to Machine Learning Algorithms.

**Comment by RUTAGENGWA-ASANTE-BRUCE** [comment, marked as answer]
A [confusion matrix](https://www.simplilearn.com/tutorials/machine-learning-tutorial/confusion-matrix-machine-learning) (or error matrix) is a specific table that is used to measure the performance of an algorithm. It is mostly used in supervised learning; in unsupervised learning, it’s called the matching matrix.

The confusion matrix has two parameters:

1. Actual
2. Predicted 
It also has identical sets of features in both of these dimensions.

Consider a confusion matrix (binary matrix) shown below:
